app/com/gu/viewer/proxy/PreviewProxy.scala

```diff
@@ -71,11 +71,10 @@ class PreviewProxy(proxyClient: ProxyClient, config: AppConfig)(implicit ec: Exe
     log.info(s"Proxy GET to preview: $url")
 
     def isLoginRedirect(response: ProxyResponse) = {
-      response.status == 303 &&
-        response.header("Location").exists(l => l == previewLoginUrl || l == "/login")
+      response.status == 303 && response.header("Location").isDefined
```

Comment:
a bit concerned that this might pick up any unrelated redirect, but then again I can't see any so can't point directly at a problem...